client/main.lua

```diff
@@ -210,11 +210,11 @@ Citizen.CreateThread(function()
                                             GarbageLocation = 1
                                             IsWorking = true
                                             SetGarbageRoute()
-                                            QBCore.Functions.Notify("You have $250,- deposit paid!")
-                                            QBCore.Functions.Notify("You have started working, location marked on GPS!")
+                                            QBCore.Functions.Notify("$250 dollar deposit was taken out of your bank account!")
```

Comment:
QBCore.Functions.Notify("The $250 deposit has been taken out of your bank account!")